src/Components/Details/Details.css

```diff
@@ -0,0 +1,66 @@
+.detailsContainer {
+  color: yellow;
+  background-color: #190738;
+}
+
+.imageContainer {
+  display: flex;
+  justify-content: space-between;
+  padding: 50px;
+}
+
+.indiImg {
+  width: 120px;
+  height: 120px;
+}
+
+.detailTitle {
+  background-color: #170630;
+  padding: 7px 20px;
+}
+
+.info {
+  border-bottom: 1px solid #170630;
+  padding: 20px;
+}
+
+.info:nth-child(even) {
+  background-color: #170630;
+}
+
+.infoLink {
+  list-style: none;
+  background-color: #190738;
+  padding: 20px;
+}
+
+.infoLink:nth-child(even) {
+  background-color: #170630;
+}
+
+.coin-info {
+  display: flex;
+  justify-content: space-between;
+}
+
+@media only screen and (max-width: 600px) {
+  .indiImg {
+    width: 100px;
+    height: 100px;
+  }
+
+  .info,
+  .infoLink {
+    padding: 10px;
+    font-size: 13px;
+  }
+
+  .detailTitle {
+    padding: 5px 10px;
+    font-size: 20px;
+  }
+
+  .imageContainer {
+    padding: 30px;
+  }
+}
```

Comment:
- [ ] In keeping with the expected design, kindly add space between the attributes and their values.

  Your Implementation | Provided Design
  :---:|:---:
  ![details-1](https://user-images.githubusercontent.com/54848363/181799571-50f49296-abb6-49d2-ae21-a6f1f0917fd1.jpg) | ![details-2](https://user-images.githubusercontent.com/54848363/181799560-3a2d7ceb-afba-4f8a-bba7-0f7fd0899c24.jpg)